tests/remote/test_firefox.py

```diff
@@ -85,6 +85,7 @@ def test_candidate_scraper(tmpdir, args, url):
     {'branch': 'mozilla-central', 'platform': 'mac'},
     {'branch': 'mozilla-central', 'platform': 'win32'},
     {'branch': 'mozilla-central', 'platform': 'win64'},
+    {'branch': 'mozilla-central', 'platform': 'win64-aarch64'},
```

Comment:
You actually missed to rename this platform name. It's causing tests to fail. 
```suggestion
    {'branch': 'mozilla-central', 'platform': 'win64-arm64'},
```